automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterFixtureSpec.scala

```diff
@@ -36,7 +36,7 @@ abstract class ClusterFixtureSpec extends fixture.FreeSpec with BeforeAndAfterAl
       ClusterName(clusterName),
       GoogleProject(googleProject),
       ServiceAccountInfo(None, None),
-      MachineConfig(),
+      RuntimeConfig.DataprocConfig(0, "", 5),
```

Comment:
not: for readability, maybe `RuntimeConfig.DataprocConfig(numberOfWorkers = 0, masterMachineType = "", ...)`